subsys/logging/log_backend_rtt.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2018 omSquare s.r.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log_backend.h>
+#include <logging/log_core.h>
+#include <logging/log_msg.h>
+#include <logging/log_output.h>
+#include <rtt/SEGGER_RTT.h>
+
+#if CONFIG_LOG_BACKEND_RTT_MODE_DROP
+
+#define DROP_MSG "\nmessages dropped: 99 \r"
```

Comment:
Hardcoded number, related to other hardcoded number.
IMHO this 99 should be replaced by stringified define.